src/UI/SlotArea.cpp

```diff
@@ -2078,9 +2076,13 @@ void cSlotAreaBrewingstand::Clicked(cPlayer & a_Player, int a_SlotNum, eClickAct
 
 
 
-void cSlotAreaBrewingstand::HandleBrewedItem(cPlayer & a_Player)
+void cSlotAreaBrewingstand::HandleBrewedItem(cPlayer & a_Player, const cItem & a_ClickedItem)
 {
-	a_Player.AwardAchievement(achBrewPotion);
+	// Check if the clicked item has damage value 0 (water bottle)
```

Comment:
-> `Award an achievement if the item is not a water bottle (is a real brewed potion)`